measurements/api/measurements.py

```diff
@@ -482,21 +469,34 @@ def list_measurements(
         mrwhere.append(sql.text("report.test_name = :test_name"))
         fpwhere.append(sql.text("test_name = :test_name"))
 
-    # TODO, we don't support filtering by confirmed filter in the fastpath, so
-    # we exclude all the fastpath measurements when filtering by confirmed is
-    # set
-    if confirmed is not None:
-        query_params["confirmed"] = confirmed
-        # TODO: coalesce false() ?
-        mrwhere.append(sql.text("measurement.confirmed = :confirmed"))
-        fpwhere.append(sql.text("false"))
+    # anomaly and confirmed: any value != TRUE is treated as FALSE
+
+    if anomaly is True:
+        mrwhere.append(sql.text("measurement.anomaly IS TRUE"))
+        fpwhere.append(sql.text("fastpath.anomaly IS TRUE"))
+
+    elif anomaly is False:
+        mrwhere.append(sql.text("measurement.anomaly IS NOT TRUE"))
+        fpwhere.append(sql.text("fastpath.anomaly IS NOT TRUE"))
```

Comment:
Given the `anomaly` is nullable, saying `anomaly = False` is not the same as `anomaly != True`.

I suggest changing this to:
```
mrwhere.append(sql.text("measurement.anomaly IS FALSE"))
```

Reply:
We are treating NULL as an absence of anomaly (hence we coalesce FALSE), so if someone is querying the API using anomaly=False we should return any row that is not TRUE

Reply:
Ok fair enough. We are pretty light on details in the spec, but I am pretty sure this is a behaviour change compared to the previous API version.

I am fine doing this, but I figured I would flag it to be sure we this is what we want.